code/__DEFINES/{yogs_defines}/ai.dm

```diff
@@ -0,0 +1,64 @@
+/** AI defines */
+
+///All AI machinery heat production is multiplied by this value
+#define AI_TEMPERATURE_MULTIPLIER 5 //Thermodynamics? No... No I don't think that's a thing. Balance so we don't use an insane amount of power to produce noticeable heat
```

Comment:
We’re not having two identically named defines files for different parts of the game. Rename the TG one

Reply:
wouldn't it be more intuitive to rename the yog exclusive define so that the tg one continues to match and won't be an issue moving forward

Reply:
no, we’re not a 1:1 TG fork

Reply:
renamed the yog ai-player defines to robots.dm to better indicate it's a yog exclusive extension to the robot defines set by tg

Reply:
Move it back to the original folder and give it a proper and descriptive name. Robots has nothing to do with its contents, and TG doesn’t have an equivalent

Reply:
i'd definitely like to avoid confusion in the file names, but robots.dm is where tg stores their AI defines, and ai.dm is reserved for simplemob ai defines. Like you said though tg doesn't have an equivalent for these defines, so doesn't that expressly mean it should be in the yog_defines folder? I can rename it to like player_ai if that's better, but i do think it should stay in the yog defines since it's exclusive to us like you said.

Reply:
Our AI defines are to do with an unrelated system compared to TGs robots.dm, ergo it should have a different name. 
Also we’re no longer requiring modularisation so you don’t have to use the yogs folder